egs/librispeech/ASR/transducer_emformer/streaming_feature_extractor.py

```diff
@@ -114,3 +105,33 @@ def _fetch_frames(self) -> None:
             frame = self.feature_extractor.get_frame(self.num_fetched_frames)
             self.feature_frames.append(frame)
             self.num_fetched_frames += 1
+
+
+class GreedySearchStream(FeatureExtractionStream):
+    def __init__(self, context_size: int) -> None:
+        """FeatureExtractionStream class for greedy search."""
+        super().__init__()
+        self.context_size = context_size
+        # For the RNN-T decoder, it contains the decoder output
+        # corresponding to the decoder input self.hyp.ys[-context_size:]
+        # Its shape is (decoder_out_dim,)
+        self.hyp: Hypothesis = None
```

Comment:
```suggestion
        self.hyp: Optional[Hypothesis] = None
```

Or you can initialize it here.